# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/runit/rockify.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

we need to export vars to be read inside init.d/system
currently we simulate sysv's stage 2
  Alejandro

--- etc/debian/1.orig	2005-01-15 08:58:06.000000000 -0300
+++ etc/debian/1	2005-02-10 12:33:09.000000000 -0300
@@ -1,10 +1,16 @@
 #!/bin/sh
 # system one time tasks
 
-PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
+{
+export PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
 
-/etc/init.d/rcS
-/etc/init.d/rmnologin
+/etc/rc.d/init.d/system start
+#/etc/rc.d/init.d/network start
+
+# write ET mark for btee
+echo -ne '\004'
+
+} 2>&1 | /sbin/btee a /var/log/init.msg
 
 touch /etc/runit/stopit
 chmod 0 /etc/runit/stopit
--- etc/debian/3.orig	2005-01-15 08:58:06.000000000 -0300
+++ etc/debian/3	2005-02-10 12:54:00.000000000 -0300
@@ -1,10 +1,11 @@
 #!/bin/sh
 exec 2>&1
 
-PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
+export PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
 
-LAST=0
-test -x /etc/runit/reboot && LAST=6
+export INIT_DOES_REBOOTHALT=1
+export RUNLEVEL=0
+test -x /etc/runit/reboot && export RUNLEVEL=6
 
 echo 'Waiting for getties to stop...'
 svwaitdown -xk -t14 /var/service/getty-*
@@ -13,4 +14,4 @@
 svwaitdown -xk -t350 /var/service/*
 
 echo 'Shutdown...'
-/etc/init.d/rc $LAST
+/etc/rc.d/init.d/system stop
--- etc/debian/ctrlaltdel.orig	2005-02-10 12:44:52.000000000 -0300
+++ etc/debian/ctrlaltdel	2005-02-10 13:25:45.000000000 -0300
@@ -4,6 +4,6 @@
 MSG="System is going down in 14 seconds..."
 
 # echo 'disabled.' ; exit
-touch /etc/runit/stopit
-chmod 100 /etc/runit/stopit && echo "$MSG" | wall
+touch /etc/runit/{stopit,reboot}
+chmod 100 /etc/runit/{stopit,reboot} && echo "$MSG" | wall
 /bin/sleep 14
